src/components/modules/SearchResultsListItem.vue

```diff
@@ -51,6 +51,21 @@
         <collection-add-to
           v-bind:item="article"
           v-bind:text="$t('add_to_collection')" />
+
+        <div v-if="article.issue.accessRights === 'OpenPublic'" class="shareArticleControl d-inline ml-1">
```

Comment:
Would be great to add `OpenPublic` as a constant somewhere and use this constant in the template.

Reply:
changed this a bit, accessRights are now per **article**